libraries/exoplayer_rtsp/src/test/java/androidx/media3/exoplayer/rtsp/RtpPacketTest.java

```diff
@@ -68,6 +68,29 @@ public final class RtpPacketTest {
       Arrays.copyOfRange(
           rtpDataWithLargeTimestamp, RtpPacket.MIN_HEADER_SIZE, rtpDataWithLargeTimestamp.length);
 
+  /*
+   10.. .... = Version: RFC 1889 Version (2)
+   ..0. .... = Padding: False
+   ...0 .... = Extension: True
```

Comment:
If the extension is true then this bit should be a 1.

Reply:
True, it was an error in copying the code. Also, I changed 1 to 0 at Marker bit.